Content.Shared/PowerCell/SharedPowerCellSystem.cs

```diff
@@ -58,7 +58,10 @@ private void OnCellRemoved(EntityUid uid, PowerCellSlotComponent component, EntR
 
     private void OnCellSlotInit(EntityUid uid, PowerCellSlotComponent component, ComponentInit args)
     {
-        _itemSlotsSystem.AddItemSlot(uid, CellSlotContainer, component.CellSlot);
+        if (_itemSlotsSystem.TryGetSlotById(uid, component.CellSlotId, out var itemSlot))
+        {
+            component.CellSlot = itemSlot;
+        }
 
         if (string.IsNullOrWhiteSpace(component.CellSlot.Name) &&
             !string.IsNullOrWhiteSpace(component.SlotName))
```

Comment:
This can result in `component.CellSlot` being null. But I'm not sure what course of action I should take in that event. I think I should throw an exception, because the real problem is that the yml is misconfigured. But I'm not sure what our best practice is in this case.